junit/src/main/java/org/kaazing/k3po/junit/rules/Latch.java

```diff
@@ -93,7 +93,7 @@ void notifyFinished() {
         switch (state) {
         case INIT:
             notifyPrepared();
-            break;
+//            break;
```

Comment:
> notifyPrepared might be needed in case the main thread is in awaitPrepared and there is an abort coming right after preparing the script. Something like this might occur in case we have an assumeTrue or assumeFalse.

Prepare Prepared comes before the test execution starts, in the k3po rule with the annotation.  So how would there be multiple threads?   I would assume prepare would block for prepared, then the test method starts getting to the assumetrue/assumefalse